cmd/logger/console.go

```diff
@@ -27,7 +27,19 @@ import (
 // in plain or json format to the standard output.
 type ConsoleTarget struct{}
 
-func (c *ConsoleTarget) send(entry logEntry) error {
+func (c *ConsoleTarget) send(e interface{}) error {
+	entry, ok := e.(logEntry)
+	if !ok {
+		if _, ok = e.(auditEntry); ok {
```

Comment:
What I meant was that `auditentry` is not being passed to `consoletarget` in `AuditLog` function, this piece of code will not be executed, even if it is, it will print in `json` format even if json flag is not enabled. So, we should just maybe return an error if the type is not `logEntry`

Reply:
I can return error since its not supported in console target